webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+
+const SRC_DIR = path.join(__dirname, '/client/src');
+
+const DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
+  output: {
+    filename: 'bundle.js',
+    path: DIST_DIR,
+  },
+  mode: 'development',
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
```

Comment:
Tried running `npm run build` on my own local machine after cloning this branch down and running `npm install`. Got this error, not sure why. Do you see this when you try to run webpack build?
<img width="725" alt="screen shot 2019-02-07 at 8 29 23 pm" src="https://user-images.githubusercontent.com/25331378/52458946-367e4d80-2b17-11e9-8ccf-30d5d68756bf.png">

Reply:
I found the issue. The `main` entry in package.json is index.jsx, which is not in my file system for this branch. It should work properly afterwards. Also, is it ideal for index.jsx to be my entry point?

Reply:
I think it would be ideal to have your server/index.js to be your entry point since you're serving everything from that file. Other than that, your changes look good, LGTM.